contrib/libdbusmenu-glib/template.py

```diff
@@ -0,0 +1,75 @@
+pkgname = "libdbusmenu-glib"
+pkgver = "16.04.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-dumper",
+    "--disable-gtk-doc",
+    "--enable-tests",
+    "--enable-static",
+    "--enable-gtk",
+    "--enable-introspection",
+    "--enable-vala",
+    "HAVE_VALGRIND_FALSE=''",
```

Comment:
this doesn't look necessary?

Reply:
removed last three lines, but instead of HAVE_VALGRIND_FALSE='' i made a patch, so with --disable-tests it wouldn't fail